src/components/reusable/notification/notification.ts

```diff
@@ -170,14 +191,16 @@ export class Notification extends LitElement {
                 class="notification-toast-close-btn"
                 kind="tertiary"
                 size="small"
-                description="close-btn"
+                description="${ifDefined(this.closeBtnDescription)}"
                 iconPosition="left"
                 @on-click="${() => this._handleClose()}"
               >
                 <kd-icon
                   slot="icon"
                   fill="#3D3C3C"
                   .icon=${closeIcon}
+                  role="img"
+                  aria-label="Close icon"
```

Comment:
No hard-coded strings, must be customizable for localization purposes. Instead of "Close icon"can we  just have "Close" text?

Reply:
Can also add a title attribute here ?